tools/cpp/runfiles/runfiles_test.cc

```diff
@@ -587,9 +587,11 @@ TEST_F(RunfilesTest, PathsFromEnvVars) {
 TEST_F(RunfilesTest, ManifestBasedRlocationWithRepoMapping_fromMain) {
   string uid = LINE_AS_STRING();
   unique_ptr<MockFile> rm(MockFile::Create(
-      "foo" + uid + ".repo_mapping",
-      {",my_module,_main", ",my_protobuf,protobuf~3.19.2",
-       ",my_workspace,_main", "protobuf~3.19.2,protobuf,protobuf~3.19.2"}));
+      "foo" + uid + "_repo_mapping",
```

Comment:
how come this got changed from `foo_uid.repo_mapping` to `foo_uid_repo_mapping` and the test didn't fail?

Reply:
The change wasn't intentional, but the only thing that matters is that the path is the target of `_repo_mapping` in the manifest, which is guaranteed by line 598.

Reply:
Dropped that change to make the diff cleaner.